tests/components/renault/test_config_flow.py

```diff
@@ -12,126 +13,198 @@
+        )
+        assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+        assert result["errors"] == {}
+
+        renault_account_1 = RenaultAccount(
+            "account_id_1",
+            websession=aiohttp_client.async_get_clientsession(hass),
+        )
+        renault_account_2 = RenaultAccount(
+            "account_id_2",
+            websession=aiohttp_client.async_get_clientsession(hass),
+        )
+
+        # Multiple accounts
+        with patch("renault_api.renault_session.RenaultSession.login"), patch(
+            "renault_api.renault_client.RenaultClient.get_api_accounts",
+            return_value=[renault_account_1, renault_account_2],
+        ), patch("renault_api.renault_account.RenaultAccount.get_vehicles"):
+            result = await hass.config_entries.flow.async_configure(
+                result["flow_id"],
+                user_input={
+                    CONF_LOCALE: "fr_FR",
+                    CONF_USERNAME: "email@test.com",
+                    CONF_PASSWORD: "test",
+                },
+            )
+
+        assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+        assert result["step_id"] == "kamereon"
+
+        # Account selected
         result = await hass.config_entries.flow.async_configure(
             result["flow_id"],
-            user_input={
-                CONF_LOCALE: "fr_FR",
-                CONF_USERNAME: "email@test.com",
-                CONF_PASSWORD: "test",
-            },
+            user_input={CONF_KAMEREON_ACCOUNT_ID: "account_id_2"},
         )
+        assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
+        assert result["title"] == "account_id_2"
+        assert result["data"][CONF_USERNAME] == "email@test.com"
+        assert result["data"][CONF_PASSWORD] == "test"
+        assert result["data"][CONF_KAMEREON_ACCOUNT_ID] == "account_id_2"
+        assert result["data"][CONF_LOCALE] == "fr_FR"
 
-    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
-    assert result["step_id"] == "kamereon"
-
-    # Account selected
-    result = await hass.config_entries.flow.async_configure(
-        result["flow_id"],
-        user_input={CONF_KAMEREON_ACCOUNT_ID: "account_id_2"},
-    )
-    assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
-    assert result["title"] == "account_id_2"
-    assert result["data"][CONF_USERNAME] == "email@test.com"
-    assert result["data"][CONF_PASSWORD] == "test"
-    assert result["data"][CONF_KAMEREON_ACCOUNT_ID] == "account_id_2"
-    assert result["data"][CONF_LOCALE] == "fr_FR"
+    assert len(mock_setup_entry.mock_calls) == 1
+
+
+async def test_config_flow_duplicate(hass: HomeAssistant):
+    """Test abort if unique_id configured."""
+    with patch(
+        "homeassistant.components.renault.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
+        await setup_renault_integration_simple(hass)
```

Comment:
It may be enough to add a MockConfigEntry.

Reply:
Should be good now